Time taken to run the queries on ./repositories/Django/thread10/ANHD-NYC-CODE_anhd-council-backend/ANHD-NYC-CODE-anhd-council-backend-d9638aa: 885.725 seconds.
Time taken to run the queries on ./repositories/Django/thread10/ASKCOS_ASKCOS/ASKCOS-ASKCOS-8bc0cee: 978.271 seconds.
Analysis failed for: ./repositories/Django/thread10/AccordBox_wagtail-tailwind-blog/AccordBox-wagtail-tailwind-blog-e7ea2b5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/AccordBox_wagtail-tailwind-blog/AccordBox-wagtail-tailwind-blog-e7ea2b5 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/AccordBox_wagtail-tailwind-blog/AccordBox-wagtail-tailwind-blog-e7ea2b5: 95.338 seconds.
Analysis failed for: ./repositories/Django/thread10/AliBigdeli_Ultimate-Django3.2-Template/AliBigdeli-Ultimate-Django3.2-Template-5891797 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/AliBigdeli_Ultimate-Django3.2-Template/AliBigdeli-Ultimate-Django3.2-Template-5891797 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/AliBigdeli_Ultimate-Django3.2-Template/AliBigdeli-Ultimate-Django3.2-Template-5891797: 93.993 seconds.
Time taken to run the queries on ./repositories/Django/thread10/Amay22_PyFolio/Amay22-PyFolio-fd946d0: 322.736 seconds.
Analysis failed for: ./repositories/Django/thread10/AvailaOSS_agube/AvailaOSS-agube-6699eca Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/AvailaOSS_agube/AvailaOSS-agube-6699eca -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/AvailaOSS_agube/AvailaOSS-agube-6699eca: 84.613 seconds.
Analysis failed for: ./repositories/Django/thread10/C3BI-pasteur-fr_ngphylogeny-django/C3BI-pasteur-fr-ngphylogeny-django-f6eca1f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/C3BI-pasteur-fr_ngphylogeny-django/C3BI-pasteur-fr-ngphylogeny-django-f6eca1f -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/C3BI-pasteur-fr_ngphylogeny-django/C3BI-pasteur-fr-ngphylogeny-django-f6eca1f: 113.04 seconds.
Time taken to run the queries on ./repositories/Django/thread10/CallMeReznov_Easy_monitor/CallMeReznov-Easy_monitor-2ab0c51: 92.477 seconds.
Analysis failed for: ./repositories/Django/thread10/City-of-Helsinki_city-infrastructure-platform/City-of-Helsinki-city-infrastructure-platform-ff0f043 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/City-of-Helsinki_city-infrastructure-platform/City-of-Helsinki-city-infrastructure-platform-ff0f043 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/City-of-Helsinki_city-infrastructure-platform/City-of-Helsinki-city-infrastructure-platform-ff0f043: 112.805 seconds.
Time taken to run the queries on ./repositories/Django/thread10/City-of-Helsinki_respa/City-of-Helsinki-respa-58421a2: 503.828 seconds.
Time taken to run the queries on ./repositories/Django/thread10/ClintonCode20_ClintonCode20/ClintonCode20-ClintonCode20-d14f2d2: 402.421 seconds.
Time taken to run the queries on ./repositories/Django/thread10/DK-Nguyen_django-social-network/DK-Nguyen-django-social-network-6061e28: 457.154 seconds.
Time taken to run the queries on ./repositories/Django/thread10/Deali-Axy_DjangoStarter/Deali-Axy-DjangoStarter-35c8e53: 2651.222 seconds.
Analysis failed for: ./repositories/Django/thread10/Edinburgh-Genome-Foundry_CUBA/Edinburgh-Genome-Foundry-CUBA-d575659 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/Edinburgh-Genome-Foundry_CUBA/Edinburgh-Genome-Foundry-CUBA-d575659 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/Edinburgh-Genome-Foundry_CUBA/Edinburgh-Genome-Foundry-CUBA-d575659: 82.901 seconds.
Analysis failed for: ./repositories/Django/thread10/FioraLove_FioraLove.github.io/FioraLove-FioraLove.github.io-8921033 Reason: Error: Command failed: codeql database create ./repositories/Django/thread10/FioraLove_FioraLove.github.io/FioraLove-FioraLove.github.io-8921033-database --language=python --source-root ./repositories/Django/thread10/FioraLove_FioraLove.github.io/FioraLove-FioraLove.github.io-8921033 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/FioraLove_FioraLove.github.io/FioraLove-FioraLove.github.io-8921033-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/FioraLove_FioraLove.github.io/FioraLove-FioraLove.github.io-8921033-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread10/FioraLove_FioraLove.github.io/FioraLove-FioraLove.github.io-8921033: 10.022 seconds.
Analysis failed for: ./repositories/Django/thread10/Fueled_django-init/Fueled-django-init-8dbac34 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/Fueled_django-init/Fueled-django-init-8dbac34 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/Fueled_django-init/Fueled-django-init-8dbac34: 87.964 seconds.
Time taken to run the queries on ./repositories/Django/thread10/GhostManager_Ghostwriter/GhostManager-Ghostwriter-8e891a1: 503.076 seconds.
Analysis failed for: ./repositories/Django/thread10/GispoCoding_ngz-geoviz/GispoCoding-ngz-geoviz-3620f46 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/GispoCoding_ngz-geoviz/GispoCoding-ngz-geoviz-3620f46 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/GispoCoding_ngz-geoviz/GispoCoding-ngz-geoviz-3620f46: 107.003 seconds.
Time taken to run the queries on ./repositories/Django/thread10/GreaterWMS_GreaterWMS/GreaterWMS-GreaterWMS-dd96e8f: 151.586 seconds.
Analysis failed for: ./repositories/Django/thread10/GunaShekar02_generator-front-to-back/GunaShekar02-generator-front-to-back-4fab92f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/GunaShekar02_generator-front-to-back/GunaShekar02-generator-front-to-back-4fab92f -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/GunaShekar02_generator-front-to-back/GunaShekar02-generator-front-to-back-4fab92f: 104.657 seconds.
Time taken to run the queries on ./repositories/Django/thread10/HuQi2018_BiSheServer/HuQi2018-BiSheServer-208e743: 811.523 seconds.
Analysis failed for: ./repositories/Django/thread10/ImperialCollegeLondon_django-drf-filepond/ImperialCollegeLondon-django-drf-filepond-ced2084 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/ImperialCollegeLondon_django-drf-filepond/ImperialCollegeLondon-django-drf-filepond-ced2084 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/ImperialCollegeLondon_django-drf-filepond/ImperialCollegeLondon-django-drf-filepond-ced2084: 170.811 seconds.
Time taken to run the queries on ./repositories/Django/thread10/LibraryOfCongress_concordia/LibraryOfCongress-concordia-4871d24: 924.364 seconds.
Time taken to run the queries on ./repositories/Django/thread10/MFOSSociety_NSP/MFOSSociety-NSP-5783498: 611.059 seconds.
Time taken to run the queries on ./repositories/Django/thread10/MH-Blog_Python/MH-Blog-Python-7662f89: 938.706 seconds.
Time taken to run the queries on ./repositories/Django/thread10/MTES-MCT_carbure/MTES-MCT-carbure-e3a4dad: 974.19 seconds.
Time taken to run the queries on ./repositories/Django/thread10/MihirSachdeva_pesticide-docker/MihirSachdeva-pesticide-docker-c674ffe: 712.291 seconds.
Time taken to run the queries on ./repositories/Django/thread10/Moagudok_Moagudok-BE/Moagudok-PJ-Moagudok_BE-189d412: 720.23 seconds.
Time taken to run the queries on ./repositories/Django/thread10/NGEET_ngt-archive/NGEET-ngt-archive-eb43849: 744.589 seconds.
Time taken to run the queries on ./repositories/Django/thread10/NikamRohan_Price-Tracker/NikamRohan-Price-Tracker-5a6df75: 788.167 seconds.
Time taken to run the queries on ./repositories/Django/thread10/No-Country_s5-04-ft-python/No-Country-s5-04-ft-python-b76407d: 644.412 seconds.
Analysis failed for: ./repositories/Django/thread10/NurbakZh_GuideLion/NurbakZh-GuideLion-ee1145d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/NurbakZh_GuideLion/NurbakZh-GuideLion-ee1145d -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/NurbakZh_GuideLion/NurbakZh-GuideLion-ee1145d: 181.309 seconds.
Analysis failed for: ./repositories/Django/thread10/OkothPius_Masoko-Ecommerce/OkothPius-Masoko-Ecommerce-220018b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/OkothPius_Masoko-Ecommerce/OkothPius-Masoko-Ecommerce-220018b -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/OkothPius_Masoko-Ecommerce/OkothPius-Masoko-Ecommerce-220018b: 148.337 seconds.
Time taken to run the queries on ./repositories/Django/thread10/OmniDB_OmniDB/OmniDB-OmniDB-ac1ae9f: 660.758 seconds.
Time taken to run the queries on ./repositories/Django/thread10/Open-CMMS_openCMMS_backend/Open-CMMS-openCMMS_backend-56511eb: 629.079 seconds.
Analysis failed for: ./repositories/Django/thread10/ProbiusOfficial_Better_StarLink/ProbiusOfficial-Better_StarLink-7905bd3 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/ProbiusOfficial_Better_StarLink/ProbiusOfficial-Better_StarLink-7905bd3 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/ProbiusOfficial_Better_StarLink/ProbiusOfficial-Better_StarLink-7905bd3: 173.204 seconds.
Analysis failed for: ./repositories/Django/thread10/Renato79_delitaly/Renato79-delitaly-9b049bb Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/Renato79_delitaly/Renato79-delitaly-9b049bb -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/Renato79_delitaly/Renato79-delitaly-9b049bb: 190.335 seconds.
Time taken to run the queries on ./repositories/Django/thread10/RugvedB_Stackoverflow-Clone/RugvedB-Stackoverflow-Clone-f2540f8: 647.257 seconds.
Analysis failed for: ./repositories/Django/thread10/Sai-Adarsh_facial-stress-using-thermal-image/Sai-Adarsh-facial-stress-using-thermal-image-538b524 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/Sai-Adarsh_facial-stress-using-thermal-image/Sai-Adarsh-facial-stress-using-thermal-image-538b524 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/Sai-Adarsh_facial-stress-using-thermal-image/Sai-Adarsh-facial-stress-using-thermal-image-538b524: 225.498 seconds.
Time taken to run the queries on ./repositories/Django/thread10/TF-Chan-Lab_panGraphViewer/TF-Chan-Lab-panGraphViewer-7ecc405: 693.149 seconds.
Analysis failed for: ./repositories/Django/thread10/TeamE9uana_DeepMush/TeamE9uana-DeepMush-ccdba0f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/TeamE9uana_DeepMush/TeamE9uana-DeepMush-ccdba0f -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/TeamE9uana_DeepMush/TeamE9uana-DeepMush-ccdba0f: 216.26 seconds.
Analysis failed for: ./repositories/Django/thread10/Tencent_CodeAnalysis/Tencent-CodeAnalysis-b673989 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/Tencent_CodeAnalysis/Tencent-CodeAnalysis-b673989 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/Tencent_CodeAnalysis/Tencent-CodeAnalysis-b673989: 204.079 seconds.
Analysis failed for: ./repositories/Django/thread10/TianPangJi_drf_admin/TianPangJi-drf_admin-d4197a7 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/TianPangJi_drf_admin/TianPangJi-drf_admin-d4197a7 -l=python -t=11 -ct=10
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m42s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/TianPangJi_drf_admin/TianPangJi-drf_admin-d4197a7-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 36.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/TianPangJi_drf_admin/TianPangJi-drf_admin-d4197a7-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/TianPangJi_drf_admin/TianPangJi-drf_admin-d4197a7: 482.344 seconds.
Analysis failed for: ./repositories/Django/thread10/Tobi-De_fuzzy-couscous/Tobi-De-fuzzy-couscous-dbafc3f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/Tobi-De_fuzzy-couscous/Tobi-De-fuzzy-couscous-dbafc3f -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/Tobi-De_fuzzy-couscous/Tobi-De-fuzzy-couscous-dbafc3f: 128.975 seconds.
Time taken to run the queries on ./repositories/Django/thread10/TonikX_ITMO_ICT_WebDevelopment_2021-2022/TonikX-ITMO_ICT_WebDevelopment_2021-2022-57870ca: 2967.987 seconds.
Analysis failed for: ./repositories/Django/thread10/TwenteCreates_aristotle/TwenteCreates-aristotle-4c0da02 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/TwenteCreates_aristotle/TwenteCreates-aristotle-4c0da02 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/TwenteCreates_aristotle/TwenteCreates-aristotle-4c0da02: 113.181 seconds.
Analysis failed for: ./repositories/Django/thread10/TwilioDevEd_appointment-reminders-django/TwilioDevEd-appointment-reminders-django-3afe5bf Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/TwilioDevEd_appointment-reminders-django/TwilioDevEd-appointment-reminders-django-3afe5bf -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/TwilioDevEd_appointment-reminders-django/TwilioDevEd-appointment-reminders-django-3afe5bf: 73.777 seconds.
Analysis failed for: ./repositories/Django/thread10/USEPA_haztrak/USEPA-haztrak-4a1f358 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/USEPA_haztrak/USEPA-haztrak-4a1f358 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/USEPA_haztrak/USEPA-haztrak-4a1f358: 86.861 seconds.
Time taken to run the queries on ./repositories/Django/thread10/UTNkar_moore/UTNkar-moore-a2d25f6: 467.718 seconds.
Time taken to run the queries on ./repositories/Django/thread10/Wolframfriele_ripe-alerts/Wolframfriele-ripe-alerts-3e4f5bb: 503.174 seconds.
Analysis failed for: ./repositories/Django/thread10/Yezz123-Archive_Django-Authentication/Yezz123-Archive-Django-Authentication-3f20766 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/Yezz123-Archive_Django-Authentication/Yezz123-Archive-Django-Authentication-3f20766 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/Yezz123-Archive_Django-Authentication/Yezz123-Archive-Django-Authentication-3f20766: 95.467 seconds.
Time taken to run the queries on ./repositories/Django/thread10/YunoHost-Apps_django_yunohost_integration/YunoHost-Apps-django_yunohost_integration-d1c609b: 430.855 seconds.
Analysis failed for: ./repositories/Django/thread10/Zakhar02_SSAD_TG_To_Web/Zakhar02-SSAD_TG_To_Web-a03c906 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/Zakhar02_SSAD_TG_To_Web/Zakhar02-SSAD_TG_To_Web-a03c906 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/Zakhar02_SSAD_TG_To_Web/Zakhar02-SSAD_TG_To_Web-a03c906: 85.927 seconds.
Analysis failed for: ./repositories/Django/thread10/ZhangDubhe_Tropical-Cyclone-Information-System/ZhangDubhe-Tropical-Cyclone-Information-System-88d16d8 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/ZhangDubhe_Tropical-Cyclone-Information-System/ZhangDubhe-Tropical-Cyclone-Information-System-88d16d8 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/ZhangDubhe_Tropical-Cyclone-Information-System/ZhangDubhe-Tropical-Cyclone-Information-System-88d16d8: 105.994 seconds.
Time taken to run the queries on ./repositories/Django/thread10/airqo-platform_AirQo-frontend/airqo-platform-AirQo-frontend-8ff3f3c: 330.493 seconds.
Time taken to run the queries on ./repositories/Django/thread10/alialaei110_IOT-MQTT-Dashboard-for-ESP32-Plant-Sensor/alialaei1-IOT-MQTT-Dashboard-for-ESP32-Plant-Sensor-818143d: 321.32 seconds.
Time taken to run the queries on ./repositories/Django/thread10/amanpro30_digital_college/amanpro30-digital_college-0ce2ac5: 367.792 seconds.
Time taken to run the queries on ./repositories/Django/thread10/arthtyagi_geddit/arthtyagi-geddit-26047f8: 351.464 seconds.
Analysis failed for: ./repositories/Django/thread10/bastianhilton_Oscar-Django/bastianhilton-Oscar-Django-dacadf3 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/bastianhilton_Oscar-Django/bastianhilton-Oscar-Django-dacadf3 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/bastianhilton_Oscar-Django/bastianhilton-Oscar-Django-dacadf3: 100.926 seconds.
Analysis failed for: ./repositories/Django/thread10/battlefield-portal-community_bfportal.gg/battlefield-portal-community-bfportal.gg-192ab44 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/battlefield-portal-community_bfportal.gg/battlefield-portal-community-bfportal.gg-192ab44 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/battlefield-portal-community_bfportal.gg/battlefield-portal-community-bfportal.gg-192ab44: 105.182 seconds.
Time taken to run the queries on ./repositories/Django/thread10/bcgov_aries-vcr/bcgov-aries-vcr-d322d07: 431.182 seconds.
Time taken to run the queries on ./repositories/Django/thread10/carlosribas_medvet/carlosribas-medvet-b45ee8f: 338.267 seconds.
Time taken to run the queries on ./repositories/Django/thread10/ceos-seo_data_cube_ui/ceos-seo-data_cube_ui-ef50e91: 560.767 seconds.
Time taken to run the queries on ./repositories/Django/thread10/chaosinventory_chaosinventory/chaosinventory-chaosinventory-6dc775f: 337.625 seconds.
Time taken to run the queries on ./repositories/Django/thread10/china-testing_python_cn_resouce/china-testing-python_cn_resouce-abd0a1f: 2143.101 seconds.
Analysis failed for: ./repositories/Django/thread10/chloe-codes1_TIL/chloe-codes1-TIL-9ad5130 Reason: Error: Command failed: codeql database create ./repositories/Django/thread10/chloe-codes1_TIL/chloe-codes1-TIL-9ad5130-database --language=python --source-root ./repositories/Django/thread10/chloe-codes1_TIL/chloe-codes1-TIL-9ad5130 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/chloe-codes1_TIL/chloe-codes1-TIL-9ad5130-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/chloe-codes1_TIL/chloe-codes1-TIL-9ad5130-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread10/chloe-codes1_TIL/chloe-codes1-TIL-9ad5130: 11.795 seconds.
Time taken to run the queries on ./repositories/Django/thread10/chohankyun_chohankyun/chohankyun-chohankyun-fca6758: 367.192 seconds.
Analysis failed for: ./repositories/Django/thread10/christianp_django-activitypub-bot/christianp-django-activitypub-bot-db79911 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/christianp_django-activitypub-bot/christianp-django-activitypub-bot-db79911 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/christianp_django-activitypub-bot/christianp-django-activitypub-bot-db79911: 96.774 seconds.
Analysis failed for: ./repositories/Django/thread10/cloudera_CML_AMP_Canceled_Flight_Prediction/cloudera-CML_AMP_Canceled_Flight_Prediction-b21c730 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/cloudera_CML_AMP_Canceled_Flight_Prediction/cloudera-CML_AMP_Canceled_Flight_Prediction-b21c730 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/cloudera_CML_AMP_Canceled_Flight_Prediction/cloudera-CML_AMP_Canceled_Flight_Prediction-b21c730: 123.574 seconds.
Analysis failed for: ./repositories/Django/thread10/cloudmesh-community_book/cloudmesh-community-book-f971bc1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/cloudmesh-community_book/cloudmesh-community-book-f971bc1 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/cloudmesh-community_book/cloudmesh-community-book-f971bc1: 145.123 seconds.
Time taken to run the queries on ./repositories/Django/thread10/clubadm_clubadm/habrasanta-habrasanta-legacy-1e46025: 361.699 seconds.
Time taken to run the queries on ./repositories/Django/thread10/codingforentrepreneurs_MVP-Landing/codingforentrepreneurs-MVP-Landing-ee652a4: 302.497 seconds.
Time taken to run the queries on ./repositories/Django/thread10/darkreactions_DRP/darkreactions-DRP-b818f44: 638.402 seconds.
Analysis failed for: ./repositories/Django/thread10/devlup-labs_gymkhana_portal/devlup-labs-gymkhana_portal-72eed02 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/devlup-labs_gymkhana_portal/devlup-labs-gymkhana_portal-72eed02 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/devlup-labs_gymkhana_portal/devlup-labs-gymkhana_portal-72eed02: 88.685 seconds.
Analysis failed for: ./repositories/Django/thread10/djaodjin_djaoapp/djaodjin-djaoapp-a2e9393 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/djaodjin_djaoapp/djaodjin-djaoapp-a2e9393 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/djaodjin_djaoapp/djaodjin-djaoapp-a2e9393: 90.814 seconds.
Time taken to run the queries on ./repositories/Django/thread10/ec18384_Predictd/ec18384-Predictd-5a87335: 441.378 seconds.
Time taken to run the queries on ./repositories/Django/thread10/echo-cool_COMP3030J-Software-Engineering-Project-Musical-Instrument-Shop/echo-cool-COMP3030J-Software-Engineering-Project-Musical-Instrument-Shop-d4cc691: 596.958 seconds.
Time taken to run the queries on ./repositories/Django/thread10/eea_copernicus_quality_tools/eea-copernicus_quality_tools-663cd17: 559.22 seconds.
Time taken to run the queries on ./repositories/Django/thread10/egypt-shops_xshop-web/egypt-shops-xshop-web-cfce222: 441.105 seconds.
Analysis failed for: ./repositories/Django/thread10/eshaan7_django-rest-durin/eshaan7-django-rest-durin-648b308 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/eshaan7_django-rest-durin/eshaan7-django-rest-durin-648b308 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/eshaan7_django-rest-durin/eshaan7-django-rest-durin-648b308: 83.27 seconds.
Time taken to run the queries on ./repositories/Django/thread10/esrg-knights_Squire/esrg-knights-Squire-97b0aa9: 453.74 seconds.
Time taken to run the queries on ./repositories/Django/thread10/fga-eps-mds_2017.2-Classificacao-de-Risco-Pediatrico/fga-eps-mds-2017.2-Classificacao-de-Risco-Pediatrico-e7cd211: 350.844 seconds.
Analysis failed for: ./repositories/Django/thread10/fga-eps-mds_2020.2-Violeta-Desenvolvimento/fga-eps-mds-2020.2-Violeta-Desenvolvimento-364e2ca Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/fga-eps-mds_2020.2-Violeta-Desenvolvimento/fga-eps-mds-2020.2-Violeta-Desenvolvimento-364e2ca -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/fga-eps-mds_2020.2-Violeta-Desenvolvimento/fga-eps-mds-2020.2-Violeta-Desenvolvimento-364e2ca: 91.927 seconds.
Analysis failed for: ./repositories/Django/thread10/flackdl_recipe_api/flackdl-recipe_api-cbc0750 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/flackdl_recipe_api/flackdl-recipe_api-cbc0750 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/flackdl_recipe_api/flackdl-recipe_api-cbc0750: 98.077 seconds.
Analysis failed for: ./repositories/Django/thread10/front-seat_engage/front-seat-engage-21b9eb7 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/front-seat_engage/front-seat-engage-21b9eb7 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/front-seat_engage/front-seat-engage-21b9eb7: 99.33 seconds.
Time taken to run the queries on ./repositories/Django/thread10/fsinfuhh_Bitpoll/fsinfuhh-Bitpoll-d9d5c28: 405.43 seconds.
Time taken to run the queries on ./repositories/Django/thread10/galacticpuzzlehunt_gph-site/galacticpuzzlehunt-gph-site-bfd9d20: 391.642 seconds.
Time taken to run the queries on ./repositories/Django/thread10/gotzl_accservermanager/gotzl-accservermanager-7d33891: 325.835 seconds.
Time taken to run the queries on ./repositories/Django/thread10/gthole_drink-stash/gthole-drink-stash-d0b4aa9: 326.681 seconds.
Analysis failed for: ./repositories/Django/thread10/hashqueue_beer_server/hashqueue-beer_server-7b471d5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/hashqueue_beer_server/hashqueue-beer_server-7b471d5 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/hashqueue_beer_server/hashqueue-beer_server-7b471d5: 97.146 seconds.
Analysis failed for: ./repositories/Django/thread10/hvitis_geodjango-rest-vue-boilerplate/hvitis-geodjango-rest-vue-boilerplate-bd825fd Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/hvitis_geodjango-rest-vue-boilerplate/hvitis-geodjango-rest-vue-boilerplate-bd825fd -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/hvitis_geodjango-rest-vue-boilerplate/hvitis-geodjango-rest-vue-boilerplate-bd825fd: 90.266 seconds.
Analysis failed for: ./repositories/Django/thread10/inosca_ebau/inosca-ebau-ca8eafb Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/inosca_ebau/inosca-ebau-ca8eafb -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/inosca_ebau/inosca-ebau-ca8eafb: 180.463 seconds.
Analysis failed for: ./repositories/Django/thread10/isb-cgc_ISB-CGC-Webapp/isb-cgc-ISB-CGC-Webapp-e784dc2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/isb-cgc_ISB-CGC-Webapp/isb-cgc-ISB-CGC-Webapp-e784dc2 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/isb-cgc_ISB-CGC-Webapp/isb-cgc-ISB-CGC-Webapp-e784dc2: 136.625 seconds.
Analysis failed for: ./repositories/Django/thread10/itdagene-ntnu_itdagene/itdagene-ntnu-itdagene-1552ff4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/itdagene-ntnu_itdagene/itdagene-ntnu-itdagene-1552ff4 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/itdagene-ntnu_itdagene/itdagene-ntnu-itdagene-1552ff4: 100.594 seconds.
Analysis failed for: ./repositories/Django/thread10/itimor_one-workflow/itimor-one-workflow-c2c3ae8 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/itimor_one-workflow/itimor-one-workflow-c2c3ae8 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/itimor_one-workflow/itimor-one-workflow-c2c3ae8: 119.472 seconds.
Analysis failed for: ./repositories/Django/thread10/ixc_django-master-password/ixc-django-master-password-2bae287 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/ixc_django-master-password/ixc-django-master-password-2bae287 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/ixc_django-master-password/ixc-django-master-password-2bae287: 96.848 seconds.
Time taken to run the queries on ./repositories/Django/thread10/jacksun01_zeus-workflow/jacksun01-zeus-workflow-2220fdf: 409.902 seconds.
Analysis failed for: ./repositories/Django/thread10/jkbrzt_cointrol/jkbrzt-cointrol-9d8ac78 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/jkbrzt_cointrol/jkbrzt-cointrol-9d8ac78 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/jkbrzt_cointrol/jkbrzt-cointrol-9d8ac78: 86.933 seconds.
Time taken to run the queries on ./repositories/Django/thread10/juancopi81_komposair/juancopi81-komposair-ee5d865: 490.518 seconds.
Analysis failed for: ./repositories/Django/thread10/knownsec_404StarLink/knownsec-404StarLink-f31f7e5 Reason: Error: Command failed: codeql database create ./repositories/Django/thread10/knownsec_404StarLink/knownsec-404StarLink-f31f7e5-database --language=python --source-root ./repositories/Django/thread10/knownsec_404StarLink/knownsec-404StarLink-f31f7e5 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/knownsec_404StarLink/knownsec-404StarLink-f31f7e5-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/knownsec_404StarLink/knownsec-404StarLink-f31f7e5-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread10/knownsec_404StarLink/knownsec-404StarLink-f31f7e5: 13.448 seconds.
Analysis failed for: ./repositories/Django/thread10/knownsec_LSpider/knownsec-LSpider-1dcdd82 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/knownsec_LSpider/knownsec-LSpider-1dcdd82 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/knownsec_LSpider/knownsec-LSpider-1dcdd82: 104.824 seconds.
Analysis failed for: ./repositories/Django/thread10/lardbit_nefarious/lardbit-nefarious-87cb6dc Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/lardbit_nefarious/lardbit-nefarious-87cb6dc -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/lardbit_nefarious/lardbit-nefarious-87cb6dc: 155.597 seconds.
Analysis failed for: ./repositories/Django/thread10/leopardslab_CrawlerX/leopardslab-CrawlerX-5f23d2e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/leopardslab_CrawlerX/leopardslab-CrawlerX-5f23d2e -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/leopardslab_CrawlerX/leopardslab-CrawlerX-5f23d2e: 115.661 seconds.
Analysis failed for: ./repositories/Django/thread10/libo-sober_CRM/libo-sober-CRM-504f78a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/libo-sober_CRM/libo-sober-CRM-504f78a -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/libo-sober_CRM/libo-sober-CRM-504f78a: 93.462 seconds.
Analysis failed for: ./repositories/Django/thread10/lorey_personal-crm/lorey-personal-crm-5994ab1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/lorey_personal-crm/lorey-personal-crm-5994ab1 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/lorey_personal-crm/lorey-personal-crm-5994ab1: 87.019 seconds.
Analysis failed for: ./repositories/Django/thread10/mohamad-liyaghi_Tsuna-Streaming/mohamad-liyaghi-Tsuna-Streaming-ebb4cac Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/mohamad-liyaghi_Tsuna-Streaming/mohamad-liyaghi-Tsuna-Streaming-ebb4cac -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/mohamad-liyaghi_Tsuna-Streaming/mohamad-liyaghi-Tsuna-Streaming-ebb4cac: 95.183 seconds.
Analysis failed for: ./repositories/Django/thread10/nautobot_nautobot-plugin-secrets-providers/nautobot-nautobot-app-secrets-providers-be90acc Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/nautobot_nautobot-plugin-secrets-providers/nautobot-nautobot-app-secrets-providers-be90acc -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/nautobot_nautobot-plugin-secrets-providers/nautobot-nautobot-app-secrets-providers-be90acc: 87.988 seconds.
Time taken to run the queries on ./repositories/Django/thread10/openplans_shareabouts-api/openplans-shareabouts-api-502f61f: 432.377 seconds.
Analysis failed for: ./repositories/Django/thread10/opinkerfi_adagios/opinkerfi-adagios-aba1144 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/opinkerfi_adagios/opinkerfi-adagios-aba1144 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/opinkerfi_adagios/opinkerfi-adagios-aba1144: 103.406 seconds.
Analysis failed for: ./repositories/Django/thread10/osc-vitap_oschub/osc-vitap-oschub-0e4d2f2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/osc-vitap_oschub/osc-vitap-oschub-0e4d2f2 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/osc-vitap_oschub/osc-vitap-oschub-0e4d2f2: 87.22 seconds.
Analysis failed for: ./repositories/Django/thread10/pablosnt_rekono/pablosnt-rekono-8e60206 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/pablosnt_rekono/pablosnt-rekono-8e60206 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/pablosnt_rekono/pablosnt-rekono-8e60206: 105.192 seconds.
Time taken to run the queries on ./repositories/Django/thread10/palomapiot_early/palomapiot-early-3234999: 383.238 seconds.
Analysis failed for: ./repositories/Django/thread10/papermerge_papermerge-core/papermerge-papermerge-core-94dfff4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/papermerge_papermerge-core/papermerge-papermerge-core-94dfff4 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/papermerge_papermerge-core/papermerge-papermerge-core-94dfff4: 91.959 seconds.
Analysis failed for: ./repositories/Django/thread10/pinax_pinax-eventlog/pinax-pinax-eventlog-8328145 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/pinax_pinax-eventlog/pinax-pinax-eventlog-8328145 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/pinax_pinax-eventlog/pinax-pinax-eventlog-8328145: 83.687 seconds.
Analysis failed for: ./repositories/Django/thread10/pinax_pinax-referrals/pinax-pinax-referrals-6ad4640 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/pinax_pinax-referrals/pinax-pinax-referrals-6ad4640 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/pinax_pinax-referrals/pinax-pinax-referrals-6ad4640: 85.652 seconds.
Analysis failed for: ./repositories/Django/thread10/proycon_LaMachine/proycon-LaMachine-206f65f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/proycon_LaMachine/proycon-LaMachine-206f65f -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/proycon_LaMachine/proycon-LaMachine-206f65f: 84.773 seconds.
Time taken to run the queries on ./repositories/Django/thread10/psun0610_TIL/psun0610-TIL-71cb135: 2272.178 seconds.
Analysis failed for: ./repositories/Django/thread10/pycasbin_django-authorization/pycasbin-django-authorization-8045ab5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/pycasbin_django-authorization/pycasbin-django-authorization-8045ab5 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/pycasbin_django-authorization/pycasbin-django-authorization-8045ab5: 89.497 seconds.
Analysis failed for: ./repositories/Django/thread10/pylixm_django-mdeditor/pylixm-django-mdeditor-1da8735 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/pylixm_django-mdeditor/pylixm-django-mdeditor-1da8735 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/pylixm_django-mdeditor/pylixm-django-mdeditor-1da8735: 97.322 seconds.
Analysis failed for: ./repositories/Django/thread10/reactive-python_reactpy-django/reactive-python-reactpy-django-c80f94d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/reactive-python_reactpy-django/reactive-python-reactpy-django-c80f94d -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/reactive-python_reactpy-django/reactive-python-reactpy-django-c80f94d: 125.48 seconds.
Time taken to run the queries on ./repositories/Django/thread10/rvdvorak_weather_board/rvdvorak-weather_board-e8f2eb9: 371.899 seconds.
Time taken to run the queries on ./repositories/Django/thread10/sabbir-mahmud_radius_server_isp/sabbir-mahmud-radius_server_isp-87c9e31: 332.264 seconds.
Time taken to run the queries on ./repositories/Django/thread10/satyarth12_Food-Delivery-API/satyarth12-Food-Delivery-API-c9fa0d1: 407.709 seconds.
Analysis failed for: ./repositories/Django/thread10/sdslabs_studyportal-nexus/sdslabs-studyportal-nexus-5d1091e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/sdslabs_studyportal-nexus/sdslabs-studyportal-nexus-5d1091e -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/sdslabs_studyportal-nexus/sdslabs-studyportal-nexus-5d1091e: 87.193 seconds.
Analysis failed for: ./repositories/Django/thread10/sergeii_swat4stats/sergeii-swat4stats-db77aac Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/sergeii_swat4stats/sergeii-swat4stats-db77aac -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/sergeii_swat4stats/sergeii-swat4stats-db77aac: 85.968 seconds.
Time taken to run the queries on ./repositories/Django/thread10/seyyedaliayati_Django-URL-Shortener/seyyedaliayati-Django-URL-Shortener-a04fbb7: 278.691 seconds.
Time taken to run the queries on ./repositories/Django/thread10/shubham1710_ByteWalk/shubham1710-ByteWalk-9f79221: 276.764 seconds.
Analysis failed for: ./repositories/Django/thread10/shubham1710_Fenice-Network/shubham1710-Fenice-Network-3a52bf0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/shubham1710_Fenice-Network/shubham1710-Fenice-Network-3a52bf0 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/shubham1710_Fenice-Network/shubham1710-Fenice-Network-3a52bf0: 72.446 seconds.
Analysis failed for: ./repositories/Django/thread10/sihyeong671_Quant_Project/sihyeong671-Quant_Project-2d0882a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/sihyeong671_Quant_Project/sihyeong671-Quant_Project-2d0882a -l=python -t=11 -ct=10
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m24s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/sihyeong671_Quant_Project/sihyeong671-Quant_Project-2d0882a-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 18.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread10/sihyeong671_Quant_Project/sihyeong671-Quant_Project-2d0882a-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/sihyeong671_Quant_Project/sihyeong671-Quant_Project-2d0882a: 238.242 seconds.
Analysis failed for: ./repositories/Django/thread10/sjm-steffann_netbox-ddns/sjm-steffann-netbox-ddns-438f6ab Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/sjm-steffann_netbox-ddns/sjm-steffann-netbox-ddns-438f6ab -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/sjm-steffann_netbox-ddns/sjm-steffann-netbox-ddns-438f6ab: 81.368 seconds.
Analysis failed for: ./repositories/Django/thread10/slott56_TigerShark/slott56-TigerShark-10e8b37 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/slott56_TigerShark/slott56-TigerShark-10e8b37 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/slott56_TigerShark/slott56-TigerShark-10e8b37: 137.65 seconds.
Time taken to run the queries on ./repositories/Django/thread10/sparcs-kaist_otlplus/sparcs-kaist-otlplus-8086f13: 397.576 seconds.
Time taken to run the queries on ./repositories/Django/thread10/speedy-net_speedy-net/speedy-net-speedy-net-0e77762: 316.401 seconds.
Time taken to run the queries on ./repositories/Django/thread10/srianant_DNN_Hyperparameter_Optimization/srianant-DNN_Hyperparameter_Optimization-9b047a6: 588.685 seconds.
Time taken to run the queries on ./repositories/Django/thread10/star4z_familytree/star4z-familytree-3a23c3a: 218.889 seconds.
Analysis failed for: ./repositories/Django/thread10/studio-YAIVERSE_Backend/studio-YAIVERSE-Backend-ad4ab75 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/studio-YAIVERSE_Backend/studio-YAIVERSE-Backend-ad4ab75 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/studio-YAIVERSE_Backend/studio-YAIVERSE-Backend-ad4ab75: 73.585 seconds.
Analysis failed for: ./repositories/Django/thread10/tads-cnat_fretex/tads-cnat-fretex-e65ae2a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/tads-cnat_fretex/tads-cnat-fretex-e65ae2a -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/tads-cnat_fretex/tads-cnat-fretex-e65ae2a: 43.209 seconds.
Time taken to run the queries on ./repositories/Django/thread10/talesmm14_Busque-o-seu-amiguinho/talesmm14-Busque-o-seu-amiguinho-0539ffe: 155.78 seconds.
Time taken to run the queries on ./repositories/Django/thread10/techhub-community_csoc_platform/techhub-community-csoc_platform-d082116: 140.712 seconds.
Time taken to run the queries on ./repositories/Django/thread10/theneurasthenicrat_whale4/theneurasthenicrat-whale4-6d40d98: 130.698 seconds.
Analysis failed for: ./repositories/Django/thread10/theodo_falco/theodo-falco-795ca0c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/theodo_falco/theodo-falco-795ca0c -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/theodo_falco/theodo-falco-795ca0c: 40.999 seconds.
Analysis failed for: ./repositories/Django/thread10/todo-web-app_todo_django_vue_pwa/todo-web-app-todo_django_vue_pwa-2d16f7b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/todo-web-app_todo_django_vue_pwa/todo-web-app-todo_django_vue_pwa-2d16f7b -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/todo-web-app_todo_django_vue_pwa/todo-web-app-todo_django_vue_pwa-2d16f7b: 29.691 seconds.
Analysis failed for: ./repositories/Django/thread10/tournesol-app_tournesol/tournesol-app-tournesol-1099bd6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/tournesol-app_tournesol/tournesol-app-tournesol-1099bd6 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/tournesol-app_tournesol/tournesol-app-tournesol-1099bd6: 80.172 seconds.
Time taken to run the queries on ./repositories/Django/thread10/tylertaewook_tutor-scheduler-django/tylertaewook-tutor-scheduler-django-5d767fe: 112.738 seconds.
Analysis failed for: ./repositories/Django/thread10/vas3k_infomate.club/vas3k-infomate.club-7814ada Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/vas3k_infomate.club/vas3k-infomate.club-7814ada -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/vas3k_infomate.club/vas3k-infomate.club-7814ada: 59.988 seconds.
Analysis failed for: ./repositories/Django/thread10/vub-ai-lab_shepherd/vub-ai-lab-shepherd-c45a3bf Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/vub-ai-lab_shepherd/vub-ai-lab-shepherd-c45a3bf -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/vub-ai-lab_shepherd/vub-ai-lab-shepherd-c45a3bf: 54.452 seconds.
Analysis failed for: ./repositories/Django/thread10/vulekamali_datamanager/vulekamali-datamanager-68c46c0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/vulekamali_datamanager/vulekamali-datamanager-68c46c0 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/vulekamali_datamanager/vulekamali-datamanager-68c46c0: 39.855 seconds.
Analysis failed for: ./repositories/Django/thread10/w4af_w4af/w4af-w4af-dcc7eb1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/w4af_w4af/w4af-w4af-dcc7eb1 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/w4af_w4af/w4af-w4af-dcc7eb1: 72.832 seconds.
Analysis failed for: ./repositories/Django/thread10/weltlink_django-quickbooks/weltlink-django-quickbooks-526c4f0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/weltlink_django-quickbooks/weltlink-django-quickbooks-526c4f0 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/weltlink_django-quickbooks/weltlink-django-quickbooks-526c4f0: 27.707 seconds.
Time taken to run the queries on ./repositories/Django/thread10/wenyuan_dj_webssh/wenyuan-dj_webssh-2a486ca: 112.691 seconds.
Analysis failed for: ./repositories/Django/thread10/wlmac_metropolis/wlmac-metropolis-f14079a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/wlmac_metropolis/wlmac-metropolis-f14079a -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/wlmac_metropolis/wlmac-metropolis-f14079a: 38.874 seconds.
Time taken to run the queries on ./repositories/Django/thread10/zbari023_E_Commerce_Store/zbari023-E_Commerce_Store-7e21beb: 107.96 seconds.
Analysis failed for: ./repositories/Django/thread10/zulip_zulip/zulip-zulip-3a03e14 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread10/zulip_zulip/zulip-zulip-3a03e14 -l=python -t=11 -ct=10
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:144
            throw new Error("Using a custom session engine, so the analysis won't be run");
                  ^

Error: Using a custom session engine, so the analysis won't be run
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:144:19)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread10/zulip_zulip/zulip-zulip-3a03e14: 59.886 seconds.
Time taken to run the queries on ./repositories/Django/thread10/zuri-training_Favicon-Gen-Team-61/zuri-training-Favicon-Gen-Team-61-0ae2a9b: 99.256 seconds.
Time taken to run the queries and generate the statistics: 49650.592 seconds.
